rfcs/text/0000-project-ACL.md

```diff
@@ -0,0 +1,117 @@
+- Required to support Project Bases Access Control
+- To limit the users view of the DCP to only the projects they have access.
+- Allow storage of controlled access data that meets FISMA and GDPR access control requirement.
+
+### User Stories
+
+1. See user stories in [HCA Project Complete Stage RFC](https://github
+.com/HumanCellAtlas/dcp-community/blob/3ce2d4e786432d16a8540da599fb2f72a89f8c5c/rfcs/text/0015-project-completion-stage.md).
+
+1. As a data submitter, I need to limit access to my project, so that I can't ensure it is version-complete before 
+ making public.
+
+1. As a data submitter, I would like to give my lab access to my project, so they can assist in validating secondary 
+ analysis results.
+
+1. As a data submitter, I would like to make my private project accessible to the public, so that other user of the DCP
+ can benefit.  
+
+1. As a data submitter, I would like to submit data that has personally identifiable information (PII) and therefore 
+ need special permissions to access.
+ 
+1. As a component of the DCP, I would like to limit a user's view to only resource they have permission to view, to 
+ prevent exposing privileged information to a users.
+
+1. As an operator of the DCP, I would like to know why a user has access to a resource, so I may reason about that 
+ whether a user is over privileged. This is helpful for maintaining a access policy of least privileges.
+
+*Share the [User Stories](https://www.mountaingoatsoftware.com/agile/user-stories) motivating this RFC.*
+
+## Scientific "guardrails" [optional]
+
+*Describe recommended or mandated review from HCA Science governance to ensure that the RFC addresses the needs of the scientific community.*
+
+## Detailed Design
+
+### Ingest
+Ingest is where a project is first created and therefore is where ACL must first be applied to project. When a user 
+creates a new project ingest will create a project resource in fusillade and assign the user as the owner. If a user 
+wants to modify an existing project, ingest must first check with fusillade to see if the user has permission. Secondary
+analysis will have permissions to submit a analysis bundle for a project on behalf of the owner.
+ 
+### Data Store
+Data store will consult fusillade before giving a user access to download a bundles in a project.
+
+### Query Service
+Query service will build an index associating project with bundles. This will be used in combination with fusillade 
+to determine if a user has access to a bundle.
+
+### Project Ownership
+
+As the owner of a project you can assign ownership or remove ownership, as well as add data to the project, 
+remove data from the project or delete the project. The owner can also make the project public.
+
+### Access levels
+Several different access levels will be defined for a project. They are:
```

Comment:
Should owners be able to physically delete, or just tombstone?

Reply:
let allow the RFC process to decide this. I'm adding it to unresolved questions.